### PR TITLE
Feat/updated store and team

### PR DESCRIPTION
**Pull request by jdsmith022**
updated the store logos on the landing page and for /webshops. A link to /webshops has been added to navBar in CO2 compensation dropdown menu.

refactor a bit: 
/src/views/webshops/Webshops.vue (aka /webshops page) now imports component /src/components/layout/Webshops.vue

this makes it easier to update the webshop logos. If srcs/components/layout/Webshops.vue is updated, then logos are updated in /src/views/webshops/Webshops.vue

team page has also been updated. LinkedIn now has option of being each person's linkedIn profile if desired

Favicon also corrected

landing page:
Screenshot 2020-10-28 at 15 16 16

new navbar link to /webshops :
Screenshot 2020-10-28 at 15 08 48

about/updated team page:
Screenshot 2020-10-28 at 15 16 30

### CODE REVIEW

**Comment by jdsmith022** [comment]
added more shops:
Screenshot 2020-10-28 at 16 25 01

**Comment by Mil0dV** [comment]
I could have mentioned this earlier, but could you move these up:
- Naif
- Willicroft
- degroenelinde 
- urban goddess

You could swap them with dimlight <-> millers oils

File: frontend/src/components/layout/Webshops.vue [hunk 1]
```diff
@@ -203,12 +203,116 @@
...
+                        image: 'shop-46',
+                        link: 'https://bergco.net/'
+                    },
+                    {
+                        image: 'shop-47',
+                        link: 'https://ruigengeroest.nl/'
+                    },
+                    {
+                        image: 'shop-52',
+                        link: 'https://fairbow.myshopify.com/'
```
Comment (Mil0dV) [inline frontend/src/components/layout/Webshops.vue:266]: => https://fairbowusa.com/

**Review by Mil0dV — CHANGES_REQUESTED** [review]
Modal/component code looks a lot cleaner like this. Just the 2 small items mentioned before and it's good2go!

**Comment by jdsmith022** [comment]
Screenshot 2020-10-29 at 08 20 46

**Pull request merged**